app-libs/oracle/src/oracle_sources/coin_gecko.rs

```diff
@@ -119,12 +119,12 @@ impl<OracleSourceInfo: Into<TradingInfo>> OracleSource<OracleSourceInfo> for Coi
 		let response = match response {
 			Ok(response) => response,
 			Err(e) => {
-				error!("coingecko execute_exchange_rate_request() failed with: {:#?}", &e);
```

Comment:
was annoyed with vertical debug prints spamming log files. removed all prety-prints in logs